src/bp/ui-admin/src/Pages/Workspace/Bots/BotItemPipeline.tsx

```diff
@@ -140,6 +152,16 @@ const BotItemPipeline: FC<Props> = ({
             </Tag>
           </Tooltip>
         )}
+        {requiresApproval() && (
+          <div className="stage-approval-btns">
+            <Button onClick={rejectStagePromotion} small={true} intent="danger">
+              Reject
+            </Button>
+            <Button onClick={approveStagePromotion} small={true} intent="success">
+              Approve
+            </Button>
```

Comment:
`={true}` is not required, also there's an object you can use instead of the textual value. In most place we set the `text` property of the button instead of a children, but that's not neceessary

`<Button text="Approve" onClick={approveStagePromotion} small intent={Intent.SUCCESS} />`